features/scroll-to-text-fragment.yml

```diff
@@ -7,16 +7,19 @@ group:
 caniuse: url-scroll-to-text-fragment
 # This feature isn't in BCD, so the status manually matches caniuse instead:
 status:
-  baseline: false
+  baseline: low
+  baseline_low_date: 2024-09-30
   support:
     chrome: "81"
     chrome_android: "81"
     edge: "83"
+    firefox: "131"
+    firefox_android: "131"
     safari: "16.1"
     safari_ios: "16.1"
 # document.fragmentDirective cannot be considered part of this feature due to
 # https://webkit.org/b/273466. TODO: Consider partial support for this feature
 # depending on https://github.com/WICG/scroll-to-text-fragment/issues/257.
-# compat_features:
-#   - api.Document.fragmentDirective
-#   - api.FragmentDirective
+compat_features:
+  - api.Document.fragmentDirective
+  - api.FragmentDirective
```

Comment:
I _think_ the right thing to do here now is:

- Add `html.elements.a.text_fragments` to `compat_features` 
- Use `compute_from: html.elements.a.text_fragments` to do the override (instead of the manual one here)

Reply:
Oh excellent! That makes things much better, thanks for tracking that down.